broadcast/fuzz/fuzz_targets/broadcast_engine_operations.rs

```diff
@@ -174,14 +167,16 @@ fn fuzz(input: FuzzInput) {
         // Create peers
         let mut peers = Vec::new();
         let mut mailboxes: BTreeMap<PublicKey, Mailbox<PublicKey, FuzzMessage>> = BTreeMap::new();
-
         for (i, &seed) in input.peer_seeds.iter().enumerate() {
+            // Create peer
             let crypto = PrivateKey::from_seed(seed);
             let public_key = crypto.public_key();
             peers.push(public_key.clone());
 
-            let (sender, receiver) = oracle.register(public_key.clone(), i as u32).await.unwrap();
```

Comment:
This would've prevented any peers from actually connecting. I fixed it to use the same channel.